torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -124,16 +111,26 @@ def _prepare(self, use_path=False, *args, **kwargs):
             if module.bias is not None:
                 module.register_parameter('_bias', nn.Parameter(module.bias.detach()))
                 module.bias = None
-            self.bias_handles.append(module.register_forward_hook(self.bias_hook))
+            self.bias_handles.append(module.register_forward_hook(BiasHook(module.parametrizations.weight[0], self.prune_bias)))
 
-    def prepare(self, model, config):
-        r"""Prepares a model, by adding the parametrizations.
+    def prepare(self, model, config, also_prune_bias=True):
```

Comment:
Does this need to be an argument for prepare? It will be cleaner to have this be part of the sparsifier definition, i,e lets specify this option during init.